artifactory/utils/commandsummary/utils.go

```diff
@@ -4,7 +4,6 @@ import (
 	"crypto/sha1" // #nosec G505 - This is only used for encoding, not security.
 	"encoding/hex"
 	"fmt"
-	clientUtils "github.com/jfrog/jfrog-client-go/utils"
 )
 
```

Comment:
Let's have the `fileNameToSha1` function located in this file, to be included in the `IndexedFilesMap` struct, beicase it is used by this struct only.